app/helpers/hosts_helper.rb

```diff
@@ -259,6 +268,9 @@ def overview_fields(host)
       ]
     ]
     fields += host_detailed_status_list(host)
+    fields += [[_("Build duration"), build_duration(host)]]
+    fields += [[_("Build errors"), (link_to("Logs from OS installer", build_errors_host_path(:id => host.id)))]] unless host.build_errors.blank?
+    fields += [[_("Token"), host.token || _("N/A")]] if User.current.admin?
```

Comment:
❤️ Nice.

Reply:
Yeah this was a pain to get, it's kinda "security" feature so only available to admins for now.